From f5aadd575bb5b858295dd936eba7d720f5c1b949 Mon Sep 17 00:00:00 2001
From: Anthony PERARD <anthony.perard@vates.tech>
Date: Fri, 8 Aug 2025 16:55:53 +0200
Subject: libxl: Remove duplicate libxl_domain_config_gen_json prototype

Since f9e681d3d1b8, the prototype of libxl_domain_config_gen_json is
generated from IDL in _libxl_types_json.h.

Fixes: f9e681d3d1b8 ("libxl: move definition of libxl_domain_config into the IDL")
Signed-off-by: Anthony PERARD <anthony.perard@vates.tech>
Acked-by: Andrew Cooper <andrew.cooper3@citrix.com>
Reviewed-by: Jason Andryuk <jason.andryuk@amd.com>

diff --git a/tools/include/libxl_json.h b/tools/include/libxl_json.h
index 260783bfdec0..3f97267eae59 100644
--- a/tools/include/libxl_json.h
+++ b/tools/include/libxl_json.h
@@ -90,7 +90,4 @@ static inline yajl_gen libxl_yajl_gen_alloc(const yajl_alloc_funcs *allocFuncs)
 
 #endif /* !HAVE_YAJL_V2 */
 
-yajl_gen_status libxl_domain_config_gen_json(yajl_gen hand,
-                                             libxl_domain_config *p);
-
 #endif /* LIBXL_JSON_H */
